_pages/mailing_list.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: toc
+title: Mailing Lists
+permalink: /mailing-lists/
```

Comment:
`-` or `_`? What's most common right now?

Reply:
Changed to `_` for consistency.